corehq/apps/cleanup/deletable_doc_types.py

```diff
@@ -28,6 +29,7 @@
     'HqDeploy': (MAIN_DB,),
     'ILSGatewayConfig': (MAIN_DB,),
     'Invitation': (settings.USERS_GROUPS_DB,),
+    'ModelActionAudit': ('auditcare',),
     'RegistrationRequest': (MAIN_DB,),
     'StandaloneTranslationDoc': (MAIN_DB,),
     'WisePillDeviceEvent': (MAIN_DB,),
```

Comment:
Not really relevant to this PR, but this is such a great list

Reply:
+1 shoutout to @orangejenny

Reply:
❤️ 

So good to see this list growing 🌻